### PR TITLE
DoND reraise interupts 

### PR DESCRIPTION
**Pull request by jenshnielsen**
so that loops of measurements can be safely stopped

### CODE REVIEW

**Comment by jenshnielsen** [comment]
@WilliamHPNielsen

File: qcodes/utils/wrappers.py [hunk 1]
```diff
@@ -263,6 +263,7 @@ def do1d(inst_set, start, stop, num_points, delay, *inst_meas):
     """
 
     # try to flush VISA buffers at the beginning of a measurement
```
Comment (WilliamHPNielsen) [inline qcodes/utils/wrappers.py]: It is slightly confusing that `interrupted` is set on the line between the buffer flushing and the comment about buffer flushing.

**Review by WilliamHPNielsen — CHANGES_REQUESTED** [review]

File: qcodes/utils/wrappers.py (hunk 1 above)
Comment (jenshnielsen) [reply qcodes/utils/wrappers.py]: Fixed

**Review by WilliamHPNielsen — APPROVED** [review]
That's what the boys want

**Pull request merged**